uPortal-groups/uPortal-groups-pags/src/main/java/org/apereo/portal/groups/pags/testers/AbstractStringTester.java

```diff
@@ -26,7 +26,7 @@ public AbstractStringTester(IPersonAttributesGroupTestDefinition definition) {
     }
 
     @Override
-    public final boolean test(IPerson person) {
+    public boolean test(IPerson person) {
```

Comment:
out of curiosity, why is this no longer `final`?

Reply:
because we can't override it or have a same method name from a parent class.

Reply:
Also final method is used only when you have a public method on a library and that you don't want that anybody extend and change the comportement.

I can't find why we have a such case here !